<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>499</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Risen xmatEdit Alpha</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QtTreePropertyBrowser" name="qttreepropertybrowser" native="true">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>674</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="flags.qrc">
      <normaloff>:/flags/gb.png</normaloff>:/flags/gb.png</iconset>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionGerman"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSkip"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
   <addaction name="menuLanguage"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About xmatEdit...</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="icon">
    <iconset resource="flags.qrc">
     <normaloff>:/flags/gb.png</normaloff>:/flags/gb.png</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionGerman">
   <property name="icon">
    <iconset resource="flags.qrc">
     <normaloff>:/flags/de.png</normaloff>:/flags/de.png</iconset>
   </property>
   <property name="text">
    <string>Deutsch</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionSkip">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Goto next 'ImageFilePath'</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtTreePropertyBrowser</class>
   <extends>QWidget</extends>
   <header>qttreepropertybrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="flags.qrc"/>
 </resources>
 <connections/>
</ui>
